mrp_bom_attribute_match/models/mrp_bom.py

```diff
@@ -124,11 +124,15 @@ def _check_component_attributes(self):
                     )
                 )
             if not all(attr in prod_attrs for attr in comp_attrs):
+                missing_attrs = rec.env["product.attribute"].browse(
+                    [attr.id for attr in set(prod_attrs) - set(comp_attrs)]
```

Comment:
Are you sure it is correct?
Seems like `missing_attrs` is actually product attributes that have no component, the opposite of what the above if-condition is checking